cpp/src/io/shp/polygon_shapefile_reader.cpp

```diff
@@ -26,16 +27,15 @@
 
 namespace {
 
-bool _reverse_winding = false;
-
 cudf::size_type read_ring(OGRLinearRing const& ring,
                           std::vector<double>& xs,
-                          std::vector<double>& ys)
+                          std::vector<double>& ys,
+                          const cuspatial::winding_order ring_order)
```

Comment:
I don't think const is necessary here.

```suggestion
                          cuspatial::winding_order ring_order)
```